Source/AIExtensions/Private/DataProviders/AIExtDataProvider_ScalableFloat.cpp

```diff
@@ -0,0 +1,17 @@
+#include "DataProviders/AIExtDataProvider_ScalableFloat.h"
+
+void UAIExtDataProvider_ScalableFloat::BindData( const UObject & owner, int32 request_id )
+{
+    FloatValue = ScalableFloat.GetValue();
+
+    // int param was encoded directly in the float value
+    IntValue = *reinterpret_cast< int32 * >( &FloatValue );
```

Comment:
why does it cast in a different way then in UAIExrAIDataProvider_GameplayAttribute::BindData?

Reply:
because I forgot to update the other code ^^